### PR TITLE
js assignments completed for leilanivquesada.git

### PR DESCRIPTION
**Pull request by leilanivquesada**
Hi, your feedback would be very much appreciated!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -33,6 +35,9 @@ console.log('hello world')
  */
 
 // TODO: ADD CODE HERE
+let firstName = "Leilani";
+let lastName = "Quesada";
+console.log(firstName + " " + lastName);
```
Comment (ethomp-dev) [inline index.js]: You could also use a template literal 🙂

File: index.js [hunk 2]
```diff
@@ -89,6 +110,8 @@ console.log('hello world')
  */
 
 // TODO: ADD CODE HERE
+myFunction = () => "here is my random string for this function";
```
Comment (ethomp-dev) [inline index.js]: This function should be called `myArrowFunction` and you should use a variable keyword when declaring a new function (i.e. `const myArrowFunction = () => {...}`)

File: index.js [hunk 3]
```diff
@@ -103,6 +126,9 @@ console.log('hello world')
  */
 
 // TODO: ADD CODE HERE
+stringToUpper = (str) => str.toUpperCase();
```
Comment (ethomp-dev) [inline index.js]: You should use a variable keyword when declaring a new function (i.e. `const stringToUpper = (str) => {...}`)

File: index.js [hunk 4]
```diff
@@ -120,3 +146,10 @@ console.log('hello world')
  */
 
 // TODO: ADD CODE HERE
+multiplyByTwo = (number) => number * 2;
```
Comment (ethomp-dev) [inline index.js]: You should use a variable keyword when declaring a new function (i.e. `const multiplyByTwo = (number) => {...}`)

File: index.js [hunk 5]
```diff
@@ -120,3 +146,10 @@ console.log('hello world')
...
+let num1 = multiplyByTwo(1);
+let num2 = multiplyByTwo(5);
+let num3 = multiplyByTwo(10);
+console.log(num1);
+console.log(num2);
+console.log(num3)
```
Comment (ethomp-dev) [inline index.js]: You could also print these values on the same line by using commas:

```js
console.log(num1, num2, num3);
```

**Review by ethomp-dev — COMMENTED** [review]
@leilanivquesada looks great! 👍 

Just a few comments...

**Review by leilanivquesada — COMMENTED** [review]
Thanks so much for the feedback! I've updated my assignment.

File: index.js [hunk 6]
```diff
@@ -110,8 +110,8 @@ console.log(myFunction());
  */
 
 // TODO: ADD CODE HERE
-myFunction = () => "here is my random string for this function";
-console.log(myFunction());
+myArrowFunction = () => "here is my random string for this function";
```
Comment (ethomp-dev) [inline index.js:113]: You should use a variable keyword when declaring `myArrowFunction`

**Review by ethomp-dev — COMMENTED** [review]
Changes look great! Just one final comment...

**Pull request merged**